lua/registers.lua

```diff
@@ -143,17 +143,25 @@ local function open_window()
 	-- Calculate the floating window size
 	local win_height = math.min(#register_lines,
 		-- If the whole buffer doesn't fit, use the size from the current line to the height
-		math.min(height - win_line - 2, math.ceil(height * 0.8 - 4)))
+		math.min(height - win_line, math.ceil(height * 0.8 - 4)))
 	local win_width = math.ceil(width * 0.8)
 
+    -- Set window at cursor position, unless the cursor is at the bottom of the buffer
+	local opts_row = 1
```

Comment:
Can you change this to:

```lua
local opts_row = 1
if win_height < 10 then
	win_height = 10
	opts_row = win_line - 11
end
```

So it's consistent with the rest of the code.

Reply:
I will 👍.
I'll submit that style change along with the scrolloff change.

Reply:
@tversteeg Sorry took a bit of time to test and deal with my Tabs config.
It should be ready now.